jupyterlab-dataops/resources/snowsql/config

```diff
@@ -0,0 +1,80 @@
+[connections]
+# *WARNING* *WARNING* *WARNING* *WARNING* *WARNING* *WARNING*
+#
+# The Snowflake user password is stored in plain text in this file.
+# Pay special attention to the management of this file.
+# Thank you.
+#
+# *WARNING* *WARNING* *WARNING* *WARNING* *WARNING* *WARNING*
+
+#If a connection doesn't specify a value, it will default to these
+#
+#accountname = defaultaccount
+#region = defaultregion
+#username = defaultuser
+#password = defaultpassword
+#dbname = defaultdbname
+#schemaname = defaultschema
+#warehousename = defaultwarehouse
+#rolename = defaultrolename
+#proxy_host = defaultproxyhost
+#proxy_port = defaultproxyport
+
+[connections.example]
+#Can be used in SnowSql as #connect example
+
+accountname = accountname
+username = username
+password = password1234
```

Comment:
???

Reply:
This is the default config option created from snowsql cli, In our case we needed to overwrite existing config due to errors with `log_file`, by changing log destination from `../snowsql_rt.log` to  `~/.snowsql/log`

Reply:
I will delete it from config